pkgs/top-level/all-packages.nix

```diff
@@ -8259,10 +8259,6 @@ in
   bazel = callPackage ../development/tools/build-managers/bazel {
     inherit (darwin) cctools;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation CoreServices Foundation;
-  };
-  bazel_jdk11 = callPackage ../development/tools/build-managers/bazel {
```

Comment:
There might not be a need for the `runJdk` parameter after this.

Reply:
Wouldn’t it still be a useful distinction to have both `runJdk` and `buildJdk`, even if they are effectively the same?